domains/asta.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Making A Random Github profile ?!",
+  
+    "owner": {
+        "username": "4STA69",
+        "email": "zaxxmo0@gmail.com"
+    },
+    "record": {
+        "TXT": "dh=aba507b296134216316553735e9c92411e17f315",
+        "CNAME": "4sta69.github.io"
```

Comment:
You can't have both CNAME and TXT, please read the [docs](https://www.is-a.dev/docs) on how you should do it for Github Pages